lighthouse-core/audits/unsized-images.js

```diff
@@ -91,14 +91,22 @@ class UnsizedImages extends Audit {
     const unsizedImages = [];
 
     for (const image of images) {
+      // Fixed images are out of document flow and won't cause layout shifts
       const isFixedImage =
         image.cssComputedPosition === 'fixed' || image.cssComputedPosition === 'absolute';
+      if (isFixedImage) continue;
 
-      if (isFixedImage || UnsizedImages.isSizedImage(image)) continue;
-      const url = URL.elideDataURI(image.src);
+      // Zero-sized images won't cause layout shift. We approve!
+      const boundingRect = image.node.boundingRect;
+      // Either the image was set to 0,0 size or a parent is not displayed.
+      const isNotDisplayed = boundingRect.width === 0 && boundingRect.height === 0;
```

Comment:
`||` ?

Reply:
I don't think that solves the CLS issue this audit is about though. If an item has 0 width but shoves text down because it has height, we're still in trouble, no?

Reply:
oof, i didn't realize elements of zero width can take up height/any space.. weird but ok

Reply:
layout is fun :)